src/Parser/Druid/Balance/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,192 @@
+import CastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import LegendaryUpgradeChecker from 'Parser/Core/Modules/Items/LegendaryUpgradeChecker';
+import LegendaryCountChecker from 'Parser/Core/Modules/Items/LegendaryCountChecker';
+import PrePotion from 'Parser/Core/Modules/Items/PrePotion';
+import EnchantChecker from 'Parser/Core/Modules/Items/EnchantChecker';
+
+import CancelledCasts from './CancelledCasts';
+import AlwaysBeCasting from './AlwaysBeCasting';
+import MoonfireUptime from './MoonfireUptime';
+import SunfireUptime from './SunfireUptime';
+import StellarFlareUptime from './StellarFlareUptime';
+import AstralPower from './AstralPower';
+import LunarEmpowerment from './LunarEmpowerment';
+import SolarEmpowerment from './SolarEmpowerment';
+import MoonSpells from './MoonSpells';
+
+class Checklist extends CoreChecklist {
+  static dependencies = {
+    castEfficiency: CastEfficiency,
+    combatants: Combatants,
+    alwaysBeCasting: AlwaysBeCasting,
+    cancelledCasts: CancelledCasts,
+    moonfireUptime: MoonfireUptime,
+    sunfireUptime: SunfireUptime,
+    stellarFlareUptime: StellarFlareUptime,
+    astralPower: AstralPower,
+    lunarEmpowerment: LunarEmpowerment,
+    solarEmpowerment: SolarEmpowerment,
+    moonSpells: MoonSpells,
+
+    legendaryUpgradeChecker: LegendaryUpgradeChecker,
+    legendaryCountChecker: LegendaryCountChecker,
+    prePotion: PrePotion,
+    enchantChecker: EnchantChecker,
+  };
+
+  rules = [
+    new Rule({
+      name: 'Always be casting',
+      description: <Wrapper><em><b>Continuously chaining casts throughout an encounter is the single most important thing for achieving good DPS as a caster</b></em>. There shoule be no delay at all between your spell casts, it's better to start casting the wrong spell than to think for a few seconds and then cast the right spell. You should be able to handle a fight's mechanics with the minimum possible interruption to your casting. Some fights (like Argus) have unavoidable downtime due to phase transitions and the like, so in these cases 0% downtime will not be possible.</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: 'Downtime',
+            check: () => this.alwaysBeCasting.downtimeSuggestionThresholds,
+          }),
+          new Requirement({
+            name: 'Cancelled Casts',
+            check: () => this.cancelledCasts.cancelledCastSuggestionThresholds,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Maintain your DoTs on the boss',
+      description: 'DoTs are a big part of your damage. You should try to keep as high uptime on them as possible.',
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+          new Requirement({
+            name: <Wrapper><SpellLink id={SPELLS.MOONFIRE_BEAR.id} icon/> uptime</Wrapper>,
```

Comment:
Moonfire... Bear? Have you tested this and it's the right spellId? I'm guessing this is like "Guardian got to the spellId first" or something...

Reply:
Yep, I tried the other one first, but only the bear one works. I don't know what the other one is for. this is the only comments I could find about it:
  // MOONFIRE = MOONFIRE_BEAR
  // MOONFIRE_CAST = MOONFIRE

Reply:
Oh, the Moonfire debuff id is called `MOONFIRE_BEAR` and the Moonfire cast id is called `MOONFIRE`. That should really be refactored, but it's not your job to do it here.